doc/cygnus-ngsi/installation_and_administration_guide/performance_tips.md

```diff
@@ -170,21 +187,28 @@ In order to calculate the appropriate capacity, just have in consideration the f
 
 [Top](#top)
 
-##<a name="section4"></a>Events TTL
-Every Flume event managed by Cygnus has associated a <i>Time-To-Live</i> (TTL), a number specifying how many times that event can be reinjected in the channel the sink got it from. Events are reinjected when a processing error occurs (for instance, the persistence system is not available, there has been a communication breakdown, etc.). This TTL has to be configured very carefully since large TTLs may lead to a quick channel capacity exhaustion, and once reached that capacity new events cannot be put into the channel. In addition, the more large is the TTL, the more will decrease the performance of the Cygnus instance since both new fresh events will have to coexist with old not processed events in the queue.
+##<a name="section4"></a>Name Mappings
+Name Mappings feature is a powerfool tool for changing the original notified FIWARE service, FIWARE service path, entity ID and type, and attributes name and type. As a side effect of this changing, Name Mappings can be used for <i>routing</i> your data, for instance by setting a common alternative FIWARE service path for two or more original service paths, all the data regarding these servivce paths will be stored under the same CKAN package.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1324/commits/5761edf332b6de1a9bbc4ae3285193574a368c85